packages/web-pixels-extension/src/schemas/pixel-events.jtd.json

```diff
@@ -364,7 +364,7 @@
       },
       "properties": {
         "currencyCode": {
-          "ref": "CurrencyCode",
+          "type": "string",
```

Comment:
Actually, why are we exposing this JSON file in our NPM package? Should we remove this? https://socket.dev/npm/package/@shopify/web-pixels-extension/files/0.3.0/src/schemas/pixel-events.jtd.json

Reply:
Hmm, yeah I think we can probably remove it. All we really need to expose are the typescript types IIRC.